spring-cloud-netflix-zuul/src/main/java/org/springframework/cloud/netflix/zuul/util/RequestContentDataExtractor.java

```diff
@@ -16,6 +16,15 @@
 
 package org.springframework.cloud.netflix.zuul.util;
 
+import static java.util.Arrays.stream;
```

Comment:
Please look at our contributing guidelines for formatting files you can import into your IDE.  We have specific ordering for imports we follow and we dont allow `.*` imports either.